src/sentry/demo/data_population.py

```diff
@@ -1,29 +1,46 @@
+import copy
 import logging
 import functools
 import random
-from typing import (
-    List,
-    Callable,
-)
+import pytz
+import time
 
+from collections import defaultdict
 from datetime import timedelta
 from django.utils import timezone
+from uuid import uuid4
+from typing import List
+
 from sentry.interfaces.user import User as UserInterface
 from sentry.models import Project
 from sentry.utils import json
-from sentry.utils.samples import random_geo, random_ip, create_sample_event
+from sentry.utils.dates import to_timestamp
+from sentry.utils.samples import (
+    random_geo,
+    random_ip,
+    create_sample_event_basic,
+    random_normal,
+)
 from sentry.utils.snuba import SnubaError
 
 
 MAX_DAYS = 2
-SCALE_FACTOR = 0.5
+SCALE_FACTOR = 0.2
```

Comment:
I am planning on doing a refactor to move some of these to settings so I don't tweak them to make things run faster

Reply:
Decided to just add it to this PR